src/worker.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var worker_threads = require('worker_threads');
```

Comment:
We are using CommonJS here, but I compile to an ES module. Therefore, I needed to use Rollup to convert it to CommonJS.

Another consideration is whether we should embed `worker.js` directly into `resolc.js` to ship a single JavaScript file. However, that approach seems a bit hacky: [Web Workers without a separate JavaScript file](https://stackoverflow.com/questions/5408406/web-workers-without-a-separate-javascript-file).
Any thoughts?

Reply:
I think we can just release both, `worker.js` and `resolc.js`

Reply:
removed worker.js